notebooks/failure-type-classification/failure_type_classifier.ipynb

```diff
@@ -27,7 +27,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        filename = f"report_local.json"
</pre><p>Should the locally saved file also have the same day.month.dashboard.job convention?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>